handler.go

```diff
@@ -44,6 +53,9 @@ func validateArguments(handler reflect.Type) (bool, error) {
 	return handlerTakesContext, nil
 }
 
+// validateReturns validates whether the arguments returned by the lambdaHabdler are valid or not. A valid lambdaHandler
+// returns a maximum of two arguments. When there are two arguments, the second argument must be of type error. When there
+// is only one argument, that one must be of type error.
```

Comment:
Probably can add reference link - https://docs.aws.amazon.com/lambda/latest/dg/go-programming-model-handler-types.html for more information.

Reply:
Added the reference